Tasks/UsePythonVersionV0/installpythonversion.ts

```diff
@@ -0,0 +1,128 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as semver from 'semver';
+import * as rest from 'typed-rest-client';
+import * as task from 'azure-pipelines-task-lib/task';
+import * as tool from 'azure-pipelines-tool-lib/tool';
+import * as osutil from './osutil';
+
+import TaskParameters from './TaskParameters';
+
+const MANIFEST_URL = 'https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json';
+const OS_VERSION = osutil._getOsVersion();
+
+interface PythonRelease {
```

Comment:
Nit: could it make sense to move all interfaces to separate file/folder?

Reply:
Fixed in https://github.com/microsoft/azure-pipelines-tasks/pull/15820/commits/03fc2213d1ac4b332a4c8225a46024cc3791819d